### DISCUSSION TITLE
Manually add possession date?

### DISCUSSION THREAD

**Discussion by modtang** [exochron/MountJournalEnhanced#74] [Ideas]
I just got Son of Galleon but my addons were disabled so I want to add today's date to the MountJournalEnhanced info but I'm not sure how to do it.

![galleon](https://user-images.githubusercontent.com/75643732/202828594-38fac66a-4e5a-4b1b-9457-884f7a156038.png)

I added it as a note but I'd like it in the same format as the others. Any ideas?

**Comment by exochron** [comment]
That's definetly feasible. I'm just not sure where to put it visually. You're probably not going to need this function regularly.
Having a button within the display would probably just disturb in most cases.
Next Option could be a chat command. Something like: `/mje set date Son of Galleon 2022-11-19`. The parser would be quite strict. As a user you have to be carefull of what you type. Otherwise it leads to the next frustrating error.
A nice compromise could be a new Option within the mount dropdown menu. But when do you show it? Only when the date is missing? What if you made a mistake at the first attempt? So show it always then to make it also editable - for all learned mounts?

Personally I would prefer the chat command. Just because I don't want people to use that feature often. 🤔
What do you think?

**Comment by modtang** [comment]
Thanks for the reply. I think a chat command could work. Maybe make it so you can only add it and not edit but have a confirmation prompt when people are adding the date to make sure they are doing it the right way or something? The addon works really well btw, it was just a derp moment on my behalf as I was on a friend's computer and didn't think about it until after I'd already learned the mount.

**Comment by exochron** [comment]
please check out version 2.21.0.
I've added a new command to do that. see the help with: /mje  :)
